YIF_Backend/Controllers/SuperAdminController.cs

```diff
@@ -329,5 +329,25 @@ public async Task<IActionResult> GetIoEAdminIdByIoEId(string ioEId)
             var result = await _superAdminService.GetIoEAdminIdByIoEId(ioEId);
             return Ok(result.Object);
         }
+
+        /// <summary>
+        /// Delete Specialty (sets specialty IsDeleted to true).
+        /// </summary>
+        /// <returns>Success message</returns>
+        /// <response code="200">Success message</response>
+        /// <response code="400">If specialty was deleted</response>
```

Comment:
are you sure you need response of 400 error?

Reply:
No. I fix it.